### ISSUE TITLE
Feature Request: Support more programs

### ISSUE THREAD

**Issue by danisztls** [OSPG/force_xdg#1]
[Arch Wiki - XDG Base Directory](https://wiki.archlinux.org/title/XDG_Base_Directory#Supported) has a long list of configuration for many programs.

**Comment by mazunki** [comment]
Yep, I'll slowly work my way through all of the ones on the wiki. Feel free to PR some solutions, or request specific ones so I can focus on them. I'll happily accept contributions provided they're sane. 

Maybe we should make a set of policies for what's okay?

**Comment by danisztls** [comment]
I'll happily submit PRs for the ones I use though I already have a system in place. So I'm thinking if it makes sense for me to adopt your solution. Will it be easier to maintain?

**Comment by mazunki** [comment]
I was already thinking about making one environment file per evil_app, I love the way you did that.

Can you push your `.config/shell/config` into `force_xdg.git/data_home/force_xdg/env.d/`, perhaps? Maybe remove the leading underscore, or is there a good reason to keep them?

By having them all in that location, it should be fairly easy to activate/deactivate different ones.

If you have a better idea, I'll be glad to hear it :)

**Comment by mazunki** [comment]
I see some of your fixes include aliases and so, I'd rather avoid them, at least in the environment files. [Look here](https://github.com/OSPG/force_xdg/issues/3) as to why.

**Comment by mazunki** [comment]
Great, I implemented all (I hope/think) environmental changes from the Arch wiki into `env.d/`. There's a holy 111 current files there now. lol

**Comment by danisztls** [comment]
> Great, I implemented all (I hope/think) environmental changes from the Arch wiki into `env.d/`. There's a holy 111 current files there now. lol

That's incredible!

**Comment by danisztls** [comment]
> I see some of your fixes include aliases and so, I'd rather avoid them, at least in the environment files. [Look here](https://github.com/OSPG/force_xdg/issues/3) as to why.

Valid reasons but the PATH hijacking also have tradeoffs, one of them is complexity.

**Comment by danisztls** [comment]
> Can you push your `.config/shell/config` into `force_xdg.git/data_home/force_xdg/env.d/`, perhaps?

I think you covered it all by implementing the envs from Arch wiki. :)

> Maybe remove the leading underscore, or is there a good reason to keep them?

It's just to indicate that it's a partial config, it's not required.
